### DISCUSSION TITLE
Craft CMS 5 - Matrix Fields are now entries results in too many not needed calls.

### DISCUSSION THREAD

**Discussion by martin-coded** [craftcms/webhooks#92] [Q&A]
In my case when a entry changes, I create a webhook so we can populate the change to the frontend. This worked really good. 

With the updated Matrix fields, which are now also entries, there is an additional call for every changed Matrix entry within the "regular" entry. Is there a certain way how I can cancel the webhook call under certain conditions, for example when the entry doesn't have a section attached?

<img width="1366" alt="Bildschirmfoto 2024-04-19 um 15 07 26" src="https://github.com/craftcms/webhooks/assets/5357688/cc48a4c9-7b9d-4917-8781-ff8ec8ff7c0d">

**Comment by mtwalsh** [comment]
@martin-coded Did you find a solution to this? Also facing this issue and just pondering options!
